src/kayenta/reducers/selectedConfig.ts

```diff
@@ -163,6 +164,12 @@ const editingMetric = handleActions(
       ...state,
       scopeName: action.payload.scopeName,
     }),
+    [Actions.UPDATE_SIGNAL_FX_METRIC_NAME]: (state: ICanaryMetricConfig, action: Action & any) => ({
```

Comment:
I was a little confused because I saw other actions in here like Datadogs update metricName, but I will move it, and I made the reducer after I made these initial actions from copying the DataDog impl.

- [ ] Move the SignalFx actions to the SignalFx reducer.